### PR TITLE
Added desc for installing nodejs, choco

### PR DESCRIPTION
**Pull request by sharmavins23**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,12 @@ Check out repository stats
 
 # Development
 
+Obviously, to start, you need to clone the repo and `npm install`. This requires
+installing [`NodeJS`](https://nodejs.org/en). You also might need
+[Chocolatey](https://chocolatey.org/install) - With Choco, you can
```
Comment (18maherc) [inline README.md:28]:
```suggestion
[Chocolatey](https://chocolatey.org/install) (follow the instructions to install for individual use; be sure to use an administrator level terminal).
```

File: README.md [hunk 2]
```diff
@@ -23,6 +23,12 @@ Check out repository stats
...
+`choco install nodejs`. The current version of Node the project uses is 22.x,
```
Comment (18maherc) [inline README.md:29]:
```suggestion
Choco allows you to run `choco install nodejs` to easily install Node. The current version of Node the project uses is 22.x,
```

File: README.md [hunk 3]
```diff
@@ -23,6 +23,12 @@ Check out repository stats
 
 # Development
 
+Obviously, to start, you need to clone the repo and `npm install`. This requires
```
Comment (18maherc) [inline README.md:26]: run `npm install` at top level??

**Review by 18maherc — CHANGES_REQUESTED** [review]
Should we just have a numbered list here for ordered step-by-step instructions??

**Comment by sharmavins23** [comment]
Changes added

**Review by 18maherc — APPROVED** [review]

**Pull request merged**